### DISCUSSION TITLE
unkonwn "?5" appear in prompt

### DISCUSSION THREAD

**Discussion by ZichengWangdrzz** [ohmyzsh/ohmyzsh#11761] [Help] [answered]
Using Wezterm as terminal.
Insatlled by  sh -c "$(curl -fsSL https://install.ohmyz.sh)"

Here is the screenshot of this unknown string.
<img width="663" alt="WechatIMG465" src="https://github.com/ohmyzsh/ohmyzsh/assets/100417105/3b74a1a3-f396-4cef-9cfc-792f882f5fca">

**Comment by rwmitchell** [comment, marked as answer]
Are you in a git repository?  ( or other version control system)

If so, you have 5 untracked files.

**Reply by ZichengWangdrzz** [reply]
I see. I will look into it and update under this post.
